hautelook/alice-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Nelmio\\Alice\\Bridge\\Symfony\\NelmioAliceBundle": ["dev", "test"],
```

Comment:
I think this one should be removed as well :)

Reply:
My bad, didn't see the nelmio/alice recipe ;)